packages/core/src/node/backend-application.ts

```diff
@@ -135,6 +136,27 @@ export class BackendApplication {
         }
     }
 
+    protected serveGzip(req: express.Request, res: express.Response, next: express.NextFunction) {
+
+        // check if browser supports it/we have the .gz equivalent of the file.
+        const acceptsEncodings  = req.acceptsEncodings();
+
+        if (acceptsEncodings.indexOf('gzip') === -1 || !fs.existsSync(`${req.url}.gz`)) {
```

Comment:
`fs.existsSync(`${req.url}.gz`)` does it ever return true? i.e.  `req.url` is something like `http://localhost:3000/bundle.js`. It cannot exist on fs.

Reply:
from what I can analyse existsSync had experimental URL support so I think this could actually work.

Reply:
it should definitely not be *sync if it runs on request.

Reply:
can you recommend a similar API? It seems we use `fs-extra` but I'm not familiar with that library @svenefftinge

Reply:
how about just `fs.exists`?

Reply:
It's deprecated in favor of `fs.stat` according to node docs @svenefftinge